tests/LSP17ContractExtension/LSP17Extendable.behaviour.ts

```diff
@@ -911,6 +911,35 @@ export const shouldBehaveLikeLSP17 = (buildContext: () => Promise<LSP17TestConte
       });
     });
 
+    describe('edge cases', () => {
+      describe('when setting less than 20 bytes as data value for the LSP17Extension data key', () => {
+        const randomSelector = ethers.utils.hexlify(ethers.utils.randomBytes(4));
+        const randomBytes10Value = ethers.utils.hexlify(ethers.utils.randomBytes(10));
+
+        const lsp17DataKey =
+          ERC725YDataKeys.LSP17.LSP17ExtensionPrefix +
+          randomSelector.substring(2) +
+          '00000000000000000000000000000000';
```

Comment:
Suggested change as as pointed out by @b00ste , to avoid literal value errors + for better readability

```suggestion
          '00'.repeat(16);
```